### PR TITLE
[ci skip] update routing guide

### PR DESCRIPTION
**Pull request by ydakuka**

### CODE REVIEW

File: guides/source/routing.md [hunk 1]
```diff
@@ -952,7 +952,7 @@ This would cause the routing to recognize paths such as:
 
 NOTE: The actual action names aren't changed by this option. The two paths shown would still route to the `new` and `edit` actions.
 
-TIP: If you find yourself wanting to change this option uniformly for all of your routes, you can use a scope.
+TIP: If you find yourself wanting to change this option uniformly for all of your routes, you can use a `scope`.
```
Comment (robin850) [inline guides/source/routing.md]: Could you remove this please ? The sentence isn't referring to the method but the word.

**Review by robin850 — COMMENTED** [review]
Looks like there aren't any missing formatting elsewhere in the guide ! 👍 If you see any that I would have missed, please update this pull request.

Could you also squash your two commits into a single one please ?

**Comment by ydakuka** [comment]
@robin850 fixed it.

**Comment by robin850** [comment]
Thanks !

**Pull request merged**